### PR TITLE
Uniform resprite - Large

### PR DESCRIPTION
**Pull request by AmoryBlaine**
Resprites all basic jumpsuits and more. Adds missing obj sprites for a few variants. Resprites misc uniforms, resprites older AmoryBlaine sprites.

### CODE REVIEW

**Comment by AmoryBlaine** [comment]
!review

**Comment by AmoryBlaine** [comment]
!wip

**Comment by AmoryBlaine** [comment]
!review

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -107,10 +107,9 @@
 
 /obj/item/clothing/under/ert
 	name = "ERT tactical uniform"
-	desc = "A short-sleeved black uniform, paired with grey digital-camo cargo pants. It looks very tactical."
-	icon_state = "ert_uniform"
-	item_state = "bl_suit"
-	worn_state = "ert_uniform"
+	desc = "High quality military fatigues, this set is in a flavour of blue used by NT's security forces, crossed with camoflague."
```
Comment (alsoandanswer) [inline code/modules/clothing/under/miscellaneous.dm]:
```suggestion
	desc = "High quality military fatigues, this set is in a flavor of blue used by NT's security forces, crossed with camouflage."
```

File: config/example/admins.txt [hunk 2]
```diff
@@ -5,4 +5,4 @@
 # Ranks can be anything defined in admin_ranks.txt           ~Carn   #
 ######################################################################
 
-# not_a_user - Admin
+amoryblaine - Host
```
Comment (alsoandanswer) [inline config/example/admins.txt]:
```suggestion
amoryblaine - Host
```

i think you forgot to comment this out for your local thing.

well, it's not your fault, I should've given you the proper config

**Comment by AmoryBlaine** [comment]
If this goes smoothly, I'll jump right into more uniform sprites. I'll be adding in a freelancer uniform, as well as figuring out how to make mercs spawn with a randomized merc uniform.

**Comment by AmoryBlaine** [comment]
!wip

**Comment by AmoryBlaine** [comment]
!review

**Comment by AmoryBlaine** [comment]
!wip

**Comment by alsoandanswer** [comment]
Yeah you gotta change the captain and centcomm gloves too cause they're supposed to be a matching set with their uniforms

dreammaker_J8dvwD6F3v

**Comment by AmoryBlaine** [comment]
> 
> 
> Yeah you gotta change the captain and centcomm gloves too cause they're supposed to be a matching set with their uniforms
> 
> dreammaker_J8dvwD6F3v

I planned to do that in a seperate glove resprite PR, but I guess I can stick in a quick one here. I don't really get why they have gloves or those hats in the first place.

**Pull request closed without merging**